src/material.cpp

```diff
@@ -45,6 +45,9 @@ void pyne::Material::norm_comp() {
 
 
 void pyne::Material::_load_comp_protocol0(hid_t db, std::string datapath, int row) {
+  // Clear current content
+  comp.clear();
```

Comment:
Is this related to the other changes in this PR????

Reply:
some merge left over.